src/main.js

```diff
@@ -36,16 +43,15 @@ function getRandomIndex(array) {
 }
 
 function generateRandomCover() {
-  coverImage.src = covers[getRandomIndex(covers)];
-  coverTitle.innerText = titles[getRandomIndex(titles)];
-  tagline1.innerText = descriptors[getRandomIndex(descriptors)];
-  tagline2.innerText = descriptors[getRandomIndex(descriptors)];
-
-  // Update current cover to match displayed cover
-  currentCover = new Cover(
-    coverImage.src, coverTitle.innerText,
-    tagline1.innerText, tagline2.innerText
-  );
+  instantiateCover(covers[getRandomIndex(covers)], titles[getRandomIndex(titles)], descriptors[getRandomIndex(descriptors)], descriptors[getRandomIndex(descriptors)]);
```

Comment:
I think we can add descriptive variables to make the function arguments more readable.